### PR TITLE
Improvements for search - react app

### PR DESCRIPTION
**Pull request by jvega190**
- Add support for exact matches using quotes
- Add support for field boosting to improve search results

https://github.com/craftercms/craftercms/issues/3160

Modify all blueprints to comment out the example.com email address (https://github.com/craftercms/craftercms/issues/4833)

### CODE REVIEW

File: config/studio/workflow/notification-config.xml [hunk 1]
```diff
@@ -9,19 +9,19 @@
     <version>2</version>
     <lang name="en"> <!-- language specific notifications -->
         <deploymentFailureNotification> <!-- list of email addresses to notify in case of publishing failure -->
-            <email>admin@example.com</email>
-            <email>admin2@example.com</email>
+<!--            <email>admin@example.com</email>-->
+<!--            <email>admin2@example.com</email>-->
```
Comment (rart) [inline config/studio/workflow/notification-config.xml:13]:
```suggestion
             <!--<email>admin@example.com</email>-->
             <!--<email>admin2@example.com</email>-->
```

File: config/studio/workflow/notification-config.xml [hunk 2]
```diff
@@ -9,19 +9,19 @@
...
         </deploymentFailureNotification>
 
         <approverEmails> <!-- list of content approver emails -->
-            <email>admin@example.com</email>
-            <email>admin2@example.com</email>
+<!--            <email>admin@example.com</email>-->
+<!--            <email>admin2@example.com</email>-->
```
Comment (rart) [inline config/studio/workflow/notification-config.xml:18]:
```suggestion
             <!--<email>admin@example.com</email>-->
             <!--<email>admin2@example.com</email>-->
```

File: config/studio/workflow/notification-config.xml [hunk 3]
```diff
@@ -9,19 +9,19 @@
...
         </approverEmails>
 
         <!-- list of email addresses to notify in case of repository merge conflict -->
         <repositoryMergeConflictNotification>
-            <email>admin@example.com</email>
-            <email>admin2@example.com</email>
+<!--            <email>admin@example.com</email>-->
+<!--            <email>admin2@example.com</email>-->
```
Comment (rart) [inline config/studio/workflow/notification-config.xml:24]:
```suggestion
             <!--<email>admin@example.com</email>-->
             <!--<email>admin2@example.com</email>-->
```

**Review by rart — CHANGES_REQUESTED** [review]
Please correct indentation of the xml changes.

**Comment by jvega190** [comment]
@rart this is ready for merge

**Review by rart — COMMENTED** [review]
Version upgrade performed.

@jvega190 changes for next?

**Review by rart — APPROVED** [review]
Version upgrade performed.

@jvega190 changes for next?

**Comment by jvega190** [comment]
Already done/merged in https://github.com/craftercms/wordify-blueprint/commit/2ce985819dd0f93034ffd79746b03fad2639e195 (react) and https://github.com/craftercms/wordify-blueprint/commit/8a19ee456698f204679ae2c28d5dbd778e3012fc (freemarker)

**Pull request merged**